### PR TITLE
Mailing lists updated.

### LINKED ISSUE #518: [Mailing Lists] Add the rest of the mailing lists to /subscribe page
**Current State of the Meshery.io site**
https://meshery.io/subscribe is missing entries for all the project mailing lists.

**Desired State of the Meshery.io site**
Have all lists available on /subscribe:

- | Meshery Developers | dev-group@meshery.io |
- | Meshery Users | user-group@meshery.io |
- | Meshery Maintainers | maintainers@meshery.io |
- | Meshery Security and Vulnerability Reports | meshery-security-vulns-reports@meshery.dev |
- | Meshery Community | community@meshery.io |

---
### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Zain-Muiz**
Signed-off-by: Zain-Muiz <zainu10863@gmail.com>

**Description**

Have listed following mailing lists on meshery.io:

| Meshery Developers | dev-group@meshery.io |
| Meshery Users | user-group@meshery.io |
| Meshery Maintainers | maintainers@meshery.io |
| Meshery Security and Vulnerability Reports | meshery-security-vulns-reports@meshery.dev |
| Meshery Community | community@meshery.io |

This PR fixes #518 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
hey @Zain-Muiz, this behavior is happening because of the hard-coding, could you please remove most of that and think of some other way to do this? One way to try could be, keeping the size of the fields fixed and then just wrap it in a smaller view.

**Comment by Zain-Muiz** [comment]
@warunicorn19 Ping !

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@Zain-Muiz good stuff, just a little nitpick, could you please just put all the inline styling into the stylesheet.

File: _includes/mailing-lists.html [hunk 1]
```diff
@@ -1,15 +1,193 @@
...
+              class="button"
+              href="https://groups.google.com/a/meshery.io/g/dev-group"
+              >Subscribe</a
+            >
+          </div>
+        </div>
+        <div
+          class="
+            flex
+            flex--alijgn-center
```
Comment (warunicorn19) [inline _includes/mailing-lists.html:35]:
```suggestion
            flex--align-center
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by Nikhil-Ladha** [comment]
Agreed @warunicorn19 !
@Zain-Muiz if I understand correctly, you gave the class names based on bootstrap styles but unfortunately we don't use bootstrap on our sites, so the class names don't make sense here. Please use a single class and define all the styles within it, which includes: `flex, alignment, background-color` and so on...
Also, since we are also using the same `Mailing-Lists` HTML as a section on the bottom of the home page,  the styles in that are also affected due to these changes. Please fix them as well.

**Comment by Zain-Muiz** [comment]
Heyy @Nikhil-Ladha @warunicorn19 , I have fixed as both of you suggested. @Nikhil-Ladha , theres a small concern though. Given the design of the mailing list, I feel it is bit big to be on our home page. Anyway, If time permits, I'll share this on the community call.

**Comment by Nikhil-Ladha** [comment]
> Heyy @Nikhil-Ladha @warunicorn19 , I have fixed as both of you suggested. @Nikhil-Ladha , theres a small concern though. Given the design of the mailing list, I feel it is bit big to be on our home page. Anyway, If time permits, I'll share this on the community call.

Have you pushed the changes already? I can't seem to find the commit 🤔 .... I agree on the design part and I think it is a good item to be discussed on the Websites Call that we have every Monday instead of the Community Call.

**Comment by warunicorn19** [comment]
Yes let's talk about this on the websites' call 😊.

**Comment by Zain-Muiz** [comment]
> > Heyy @Nikhil-Ladha @warunicorn19 , I have fixed as both of you suggested. @Nikhil-Ladha , theres a small concern though. Given the design of the mailing list, I feel it is bit big to be on our home page. Anyway, If time permits, I'll share this on the community call.
> 
> Have you pushed the changes already? I can't seem to find the commit 🤔 .... I agree on the design part and I think it is a good item to be discussed on the Websites Call that we have every Monday instead of the Community Call.

No I didnt commit, thought would ask before that! Yeah, right, Websites will be the right place, Lets keep it for monday ! Ill commit my changes in some time and ping both of you then. Maybe any of you can have a look, if you get time before that.

**Comment by Zain-Muiz** [comment]
@warunicorn19 @Nikhil-Ladha .

P. S : will resolve conflicts once we come to a conclusion on the design :)

**Comment by Zain-Muiz** [comment]
Ping ! @warunicorn19

**Review by warunicorn19 — APPROVED** [review]
Woohoo, good stuff @Zain-Muiz 🎉, thanks for seeing this to the end. LGTM✌️.

**Review by Nikhil-Ladha — APPROVED** [review]
Looks good!

**Pull request merged**